src/Grok.Net/Grok.cs

```diff
@@ -38,21 +39,34 @@ public Grok(string grokPattern)
         /// </summary>
         /// <param name="grokPattern">The Grok pattern to use.</param>
         /// <param name="customPatterns">A stream containing custom patterns.</param>
-        public Grok(string grokPattern, Stream customPatterns)
-            : this(grokPattern)
+        public Grok(string grokPattern, Stream customPatterns) : this(grokPattern)
```

Comment:
Please don't change the defined code-style conventions. 
`: this(grokPattern)` should be at the new line

Reply:
Crap, sorry! Will change :)